inference-engine/ie_bridges/python/src/openvino/offline_transformations/CMakeLists.txt

```diff
@@ -0,0 +1,66 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set(TARGET_NAME "offline_transformations_api")
+
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PYTHON_BRIDGE_OUTPUT_DIRECTORY}/offline_transformations)
+
+file(GLOB SOURCE
+        ${CMAKE_CURRENT_SOURCE_DIR}/offline_transformations_api.pyx
+        ${CMAKE_CURRENT_SOURCE_DIR}/offline_transformations_api_impl.cpp)
+
+set_source_files_properties(${SOURCE} PROPERTIES CYTHON_IS_CXX ON)
+
+# create target
+
+cython_add_module(${TARGET_NAME} ${SOURCE})
+set(INSTALLED_TARGETS ${TARGET_NAME})
+add_dependencies(${TARGET_NAME} ie_api)
+
+if(COMMAND ie_add_vs_version_file)
+    foreach(target IN LISTS INSTALLED_TARGETS)
+        ie_add_vs_version_file(NAME ${target}
+                               FILEDESCRIPTION "Offline Transformatoins Python library")
+    endforeach()
+endif()
+
+if(TARGET offline_transformations)
+    list(APPEND InferenceEngine_LIBRARIES offline_transformations)
+else()
+    list(APPEND InferenceEngine_LIBRARIES IE::offline_transformations)
+endif()
+
+target_include_directories(${TARGET_NAME} PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}" "${CMAKE_CURRENT_SOURCE_DIR}/../inference_engine")
+target_link_libraries(${TARGET_NAME} PRIVATE ${InferenceEngine_LIBRARIES})
+
+# Compatibility with python 2.7 which has deprecated "register" specifier
+if(CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
+    target_compile_options(${TARGET_NAME} PRIVATE "-Wno-error=register")
+endif()
+
+# perform copy
+add_custom_command(TARGET ${TARGET_NAME}
+    POST_BUILD
+    COMMAND ${CMAKE_COMMAND} -E copy ${PYTHON_BRIDGE_SRC_ROOT}/src/openvino/offline_transformations/__init__.py ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/__init__.py
+)
+
+# creates a folder in openvino directory and a symlink to benchmark
+# inside bin directory for developers for running python benchmark_app
+if(UNIX)
+    add_custom_command(TARGET ${TARGET_NAME}
+        POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E make_directory ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/../tools
+)
+    file(COPY ${OpenVINO_MAIN_SOURCE_DIR}/tools/benchmark DESTINATION ${CMAKE_LIBRARY_OUTPUT_DIRECTORY}/../tools/)
+endif()
```

Comment:
I'm not sure that these ```if-endif``` block of code is needed.

Reply:
removed.